src/Resources/public/js/autocomplete-create.js

```diff
@@ -95,6 +95,17 @@ function initCreateEntityAjaxForm(field_name, select_id) {
     });
 }
 
+(function($) {
```

Comment:
As the function is globally defined, it should be defined in the main JS file (easyadmin-extension.js). Please move it.

Reply:
Example
```
    function serializeForm(form) {
        var formData = new FormData();
        var params = form.serializeArray();
        $.each(params, function (i, val) {
            formData.append(val.name, val.value);
        });
        return formData;
    };
```

Reply:
Yes it looks more simple to me (but i'm not a reference for JS). And I think it's good to put the function in the lain script => it can be used in others contexts than entity-modal